.gitignore

```diff
@@ -482,3 +482,7 @@ lib/ojdbc.jar
 
 # generated during release process
 CHANGELOG.html
+
+
```

Comment:
Remove this empty line. (because of consistency with the other lines in the file)